samples/amber.cc

```diff
@@ -431,9 +432,13 @@ int main(int argc, const char** argv) {
                                      options.buffer_to_dump.end());
   }
 
-  if (!options.image_filename.empty()) {
+  // Use default frame buffer name when not specified.
+  while (options.image_filenames.size() > options.fb_names.size())
```

Comment:
Should we check if this is going to be more then 1 and, if so, log an error an exit? Folks may not expect to get the same framebuffer written out n times.